src/mark.ts

```diff
@@ -1,8 +1,6 @@
-import { isGuardianDomain } from './domain';
-
 export const mark = (label: string) => {
 	window.performance?.mark?.(label);
-	if (!isGuardianDomain() && process.env.NODE_ENV !== 'test') {
+	if (process.env.NODE_ENV !== 'test') {
```

Comment:
Where are we setting `process.env.NOVE_ENV` to `test`? I can only see where we set it to `production` and to `development`

Reply:
jest does it automatically